content/en/blog/_posts/2022-04-28-cni-version-related-breaking-change.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: blog
+title: "Breaking changes related to CNI version in popular container runtimes"
+date: 2022-04-28T12:00:00+0000
+slug: cni-version-related-breaking-change-k8s-1.24
+---
+
+**Authors:** Mike Brown (IBM), Rolfe Dlugy-Hegwer (Red Hat), and others.
+
+---
+
+_TL;DR: Before you upgrade to Kubernetes 1.24, if you use a CNI network plugin
+that is set up to use Container Network Interface (CNI) versions earlier than
+v1.0.0 and you plan to use any of the following container runtimes, you must
+specify the CNI version. Otherwise, your network will break._
+
+This issue applies to the following container runtimes:
+- **containerd** v1.6.0 and later
+- **CRI-O** 1.24 and later
+
+Here’s an overview of the typical steps to perform on each node:
+
+1. [Safely drain and cordon the node](/docs/tasks/administer-cluster/safely-drain-node/).
+2. After stopping your container runtime and kubelet services, perform the
+following upgrade operations:
+  - If you’re running CNI plugins, upgrade them to the latest version.
+  - If you’re using non-CNI plugins, replace them with CNI plugins. Use the
+  latest version of the plugins.
+  - Update the plugin configuration file to specify/match the version of the CNI
+  specification that the plugin supports.
+  - For `containerd`, ensure that you have installed the latest version (v1.0.0
+  or later) of the CNI loopback plugin.
+  - Upgrade node components (for example, the kubelet) to Kubernetes 1.24
+  - Upgrade to or install the most current version of the container runtime.
+3. Bring the node back into your cluster by restarting your container runtime
+and kubelet. Uncordon the node.
+
+If you performed the upgrade without following these steps and your network is
+broken, see the following [Troubleshooting](#troubleshooting) section.
+
+## Explanation
+
+CNI enables Kubernetes 1.24 networking. The given container runtimes support the
```

Comment:
> CNI enables Kubernetes 1.24 networking.

I'm wary of implying that CNI doesn't also enable for Kubernetes 1.21, 1.22, 1.23 etc.

Reply:
@mikebrow What do you think about rephrasing this as:
```suggestion
Networking in Kubernetes 1.24 requires CNI. The given container runtimes support the
```

Reply:
I think @sftim 's comment is more to ok what about the other releases... 

suggest removing the version number from the sentence...

Reply:
Removed `1.24` from the current phrase. Thanks.